resolve-workgroup/decisions/AYD-RE-001-Master_timeline.md

```diff
@@ -0,0 +1,54 @@
+# AYD-RE-001 Generating master timeline from multiple timeline variants
+
+- **Status:** Proposed
+- **Impact:**  Medium
+- **Contributors:**:
+  - YNPUT Team <team@ynput.io>
+  - Jakub Jezek <jakub@ynput.io>
+  - Tony Dorfmeister  (Bepic) @tweak-wtf
+  - Jiri Sindelar (Dazzle Pictures) @jrsndl
+- **Approved by:** -
+
+## Background
+
+NLE projects are usually producing multiple version of the same timeline. Either they need to produce different lengths or different resolution. Resources for these versions are usually shared. We need to be able to generate a master timeline from multiple timeline variants. This master timeline should contain shared clips in the longest durations. Clips in the master timeline are then ideal for publishing into the VFX pipeline.
+
+## Options
+
+### Option 1
+
+Using OpenTimelineIO to generate master timeline from multiple timeline variants.
+
+#### Pros
+
+- Abstraction layer for multiple NLEs
+- Resolve already has OTIO support in the latest version
+
+#### Cons
+
+- OTIO is not yet production ready in Resolve in regards of re-timing and speed changes
```

Comment:
Note that there are some known bugs/crashes on importing `OpenTimelineIO` into Resolve/Fusion, in particular on Linux.

See:
> BTW using Python OTIO modules from inside of Resolve on Rocky Linux for just about anything other than OTIO frame to timecode conversions will instant crash Resolve.
[source](https://discord.com/channels/793508729785155594/793510371323019314/1160633761977860166)